### DISCUSSION TITLE
Heuristic evaluation criteria Ερώτημα Γ2

### DISCUSSION THREAD

**Discussion by afoscolou** [sdy60/syllabus#47] [Q&A] [answered]
Σχετικά με την αρχική αξιολόγηση από ειδικούς της ερώτησης Γ2, θα ήθελα να προτείνω να γίνει συνεργασία μεταξύ των ομάδων της ΘΕ που έχουν σχηματιστεί **ανά δύο** και να ανταλλάξουν την ίδια λίστα σημείων αξιολόγησης, ώστε να υπάρχει κοινό ενδιαφέρον και εξοικείωση με τα ερωτήματα. Θα μπορούσε να φτιαχτεί και ένα κοινό ερωτηματολόγιο (αν αυτό είναι αποδεκτό και επιτρεπτό) για όλες τις ομάδες, πχ με βάση τα 10 σημεία του Jacob Nielsen, για να υπάρχει μεγαλύτερη ομοιογένεια και τυποποίηση, ώστε να υπάρχουν τα ίδια standards για όλους.

Επίσης, πάνω σε ποιο υπόδειγμα της σχεδίασης θα γίνει η αξιολόγηση;  Στο πρωτότυπο χαμηλής πιστότητας που περιλαμβάνει όλο το σενάριο ή σε αυτά υψηλής πιστότητας, τα οποία όμως αναφέρονται σε ένα βήμα του σεναρίου;

**Comment by epidrome** [comment, marked as answer]
βλέπω δύο ερωτήματα, καλύτερα να γίνει σε νέο νήμα συζήτηση, εδώ σχολιάζω το πρώτο μέρος:

> Σχετικά με την αρχική αξιολόγηση από ειδικούς της ερώτησης Γ2, θα ήθελα να προτείνω να γίνει συνεργασία μεταξύ των ομάδων της ΘΕ που έχουν σχηματιστεί ανά δύο 

αυτό είναι το ερώτημα!

> και να ανταλλάξουν την ίδια λίστα σημείων αξιολόγησης, ώστε να υπάρχει κοινό ενδιαφέρον και εξοικείωση με τα ερωτήματα. Θα μπορούσε να φτιαχτεί και ένα κοινό ερωτηματολόγιο (αν αυτό είναι αποδεκτό και επιτρεπτό) για όλες τις ομάδες, πχ με βάση τα 10 σημεία του Jacob Nielsen, για να υπάρχει μεγαλύτερη ομοιογένεια και τυποποίηση, ώστε να υπάρχουν τα ίδια standards για όλους.

αυτό δεν γίνεται γιατί κάθε ομάδα έχει μάλλον πολύ διαφορετικές παραδοχές και πεδίο εφαρμογής, τα οποία απαιτούν αναζήτησή-σύνθεση. με την υιοθέτηση γενικών κανόνων σχεδίασης τύπου Νίλσεν το ερώτημα πλέον δεν έχει νόημα.
